lib/vagrant-openshift/action/install_openshift3.rb

```diff
@@ -29,6 +29,14 @@ def call(env)
           sudo(env[:machine], %{
 set -x
 
+#systemctl enable firewalld
```

Comment:
@jwforres firewalld is currently disabled.  I need to track down why.  I was just adding these for when we do enable it on these vms.